docs/machine-learning/tutorials/image-classification-model-builder.md

```diff
@@ -0,0 +1,247 @@
+1. In the **Name** text box, type "LandUse".
+1. Select **Next**.
+1. In the "Additional Information" window, click **Create** to accept the default target framework.
+
+## Prepare and understand the data
+
+> [!NOTE]
+> The data for this tutorial is from:
+>
+> - Eurosat: A novel dataset and deep learning benchmark for land use and land cover classification. Patrick Helber, Benjamin Bischke, Andreas Dengel, Damian Borth. IEEE Journal of Selected Topics in Applied Earth Observations and Remote Sensing, 2019.
+> - Introducing EuroSAT: A Novel Dataset and Deep Learning Benchmark for Land Use and Land Cover Classification. Patrick Helber, Benjamin Bischke, Andreas Dengel. 2018 IEEE International Geoscience and Remote Sensing Symposium, 2018.
+
+The EuroSAT dataset contains a collection of satellite images divided into ten categories (rural, industrial, river, etc.). The original dataset contains 27,000 images. For convenience, this tutorial only uses 2,000 of those images.
+
+  ![Dataset](./media/image-classification-model-builder/dataset-images.png)
+
+1. Download the subset of the [EuroSAT dataset](https://github.com/dotnet/machinelearning-samples/blob/master/samples/modelbuilder/ImageClassification_Azure_LandUse/assets.zip) and save it anywhere on your computer.
+1. Unzip it.
+
+## Create a Model Builder config file
+
+When first adding Model Builder to the solution it will prompt you to create an `mbconfig` file. The `mbconfig` file keeps track of everything you do in Model Builder to allow you to reopen the session.
+
+1. In Solution Explorer, right-click the **LandUse** project, and select **Add > Machine Learning Model...**.
+1. In the dialog, name the Model Builder project **LandUse**, and click **Add**.
+
+## Choose a scenario
+
+![Model Builder Scenarios](../media/model-builder-scenarios.png)
+
+To train your model, you need to select from the list of available machine learning scenarios provided by Model Builder.
+
+1. For this sample, the scenario is image classification. In the scenario step of the Model Builder tool, select the **Image Classification** scenario.
+
+## Select an enviornment
+
+Model Builder can run the training on different environments depending on the scenario that was selected.
+
+1. Select "Azure" as your environment and click the **Set up workspace** button.
+
+### Create experiment in Azure
+
+An Azure Machine Learning experiment is a resource that needs to be created before running Model Builder training on Azure.
+
+The experiment encapsulates the configuration and results for one or more machine learning training runs. Experiments belong to a specific workspace. The first time an experiment is created, its name is registered in the workspace. Any subsequent runs - if the same experiment name is used - are logged as part of the same experiment. Otherwise, a new experiment is created.
+
+1. In the Create New Experiment dialog, choose your subscription from the **Subscription** dropdown.
```

Comment:
Since it's only 1 step, we can remove the number.

```suggestion
In the Create New Experiment dialog, choose your subscription from the **Subscription** dropdown.
```